CuratedContent/EconomyOfOpenSourceSW.md

```diff
@@ -0,0 +1,28 @@
+# A look at the economic forces in Open Source Software
+
+The Slate Article *The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?* looks at the economics driving the Open Source movement and how that is putting those utilizing that software without investing in it at risk.
+
+Resource Information | Details
+:--- | :--- 
+Resource Name | "The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?"
+Website | https://www.vice.com/en_us/article/43zak3/the-internet-was-built-on-the-free-labor-of-open-source-developers-is-that-sustainable
+Focus | Funding open source software development.
+
+Much of modern HPC software is underpinned by Open Source - be that the [Linux Kernel](https://github.com/torvalds/linux), [Tensorflow](https://github.com/tensorflow/tensorflow), [SuperLU](https://github.com/xiaoyeli/superlu), or others. Yet treating the availability of that software as an externality means taking on the risk that said software has flaws that may impair your own work.
+
+Daniel Oberhaus explores the problems in this model using the now infamous "Heartbleed Bug" tracing it's introduction into openSSL through to the final discovery and fix.
+Along the way he looks at the cost estimates for dealing with the bug ([~500Million](https://www.eweek.com/security/heartbleed-ssl-flaw-s-true-cost-will-take-time-to-tally)) and compares that to the development budget for openSSL at the time ([$2000/year](http://veridicalsystems.com/blog/of-money-responsibility-and-pride)).
```

Comment:
Missing dollar sign in front of *500 Million* as well as space between *500* and *Million.

Reply:
@prwolfe sorry if my comments feel like overkill...still getting hang of this for CC content.

Reply:
Done.